### ISSUE TITLE
Testing the example

### ISSUE THREAD

**Issue by alexjp** [ethanresnick/json-api#9]
Hi,

When i test the example, currently it gives me this error :

```
/tmp/json-api-example/src/index.js:25
  , Controller = new API.controllers.API(registry);
                 ^
TypeError: undefined is not a function
    at Object.<anonymous> (/tmp/json-api-example/src/index.js:25:18)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
```

**Comment by ethanresnick** [comment]
Thanks for catching this, Alex! 

I think the issue was that the example repo's `package.json` was still loading in an earlier version of the library. I've fixed that now! Can you run `npm update` and let me know if you're still having an issue?

Thanks!

**Comment by alexjp** [comment]
Hi,

Your welcome. I tried it again and that part worked.
But now i have this error :
```
Starting up! Visit 127.0.0.1:3000 to see the docs.
^[[15~[TypeError: Cannot read property 'find' of undefined] 'TypeError: Cannot read property \'find\' of undefined\n    at module.exports (/tmp/json-api-example/node_modules/json-api/build/src/steps/do-query/do-get.js:28:19)\n    at callee$2$0$ (/tmp/json-api-example/node_modules/json-api/build/src/controllers/API.js:139:24)\n    at tryCatch (/tmp/json-api-example/node_modules/json-api/node_modules/babel-runtime/regenerator/runtime.js:56:40)\n    at GeneratorFunctionPrototype.invoke (/tmp/json-api-example/node_modules/json-api/node_modules/babel-runtime/regenerator/runtime.js:201:22)\n    at onFulfilled (/tmp/json-api-example/node_modules/json-api/node_modules/co/index.js:64:19)\n    at _fulfilled (/tmp/json-api-example/node_modules/q/q.js:794:54)\n    at self.promiseDispatch.done (/tmp/json-api-example/node_modules/q/q.js:823:30)\n    at Promise.promise.promiseDispatch (/tmp/json-api-example/node_modules/q/q.js:756:13)\n    at /tmp/json-api-example/node_modules/q/q.js:516:49\n    at flush (/tmp/json-api-example/node_modules/q/q.js:110:17)' undefined
```

Maybe something badly configured in my end ? 

It does responds the json though :
``{"links":{"self":"http://127.0.0.1:3000/schools"},"data":[]}``

**Comment by ethanresnick** [comment]
Hey @alexjp,

Nope, it's not a misconfiguration on your end; just a dumb bug in the example. I tried to [simplify the express routing lines](https://github.com/ethanresnick/json-api-example/commit/5b0f4b6841512955366a2169fd0b89aea2c33ea9#diff-1fdf421c05c1140f6d71444ea2b27638L47) the other day and, in so doing, I accidentally set it up such that requests for static files (like the favicon) were getting passed to the json-api library. That's what was causing the error. But it's [fixed now](https://github.com/ethanresnick/json-api-example/commit/98d4ff09f1fd2e74c7e7700543898061f4a46b63) and I've opened [an issue](https://github.com/ethanresnick/json-api/issues/10) to make the library handle invalid requests like this more gracefully in the future.

So, if you `git clone` the example again, everything should work!
